src/css/docs-prose.css

```diff
@@ -15,6 +15,7 @@
 }
 
 /* make links darker in Note to improve color contrast */
+.dr-ui--overview-header .link,
```

Comment:
Oh this is a tricky one as it will impact the OverviewHeader impact in the dark theme.:

<img width="605" alt="2021-09-21 at 10 29 AM" src="https://user-images.githubusercontent.com/2180540/134190028-37547d1c-d0d0-46d8-8a9d-1cd8d938213f.png">


The catalog site is using the `theme` prop of OverviewHeader, let's adjust that instead of making a global CSS change here:

https://raw.githubusercontent.com/mapbox/dr-ui/assembly-v1-catalog/docs/src/pages/components/index.md

(see `theme: bg-purple-faint`, we could remove this line or play around with other OverviewHeader settings to improve the theme.)

Reply:
I LOVE IT!